.standard-GEM.md

```diff
@@ -37,7 +37,7 @@ The topic `standard-GEM` must be added. Other topics like `genome-scale-models`,
 Topics are not copied from `standard-GEM`, so they need to be added manually.
 
 - [ ] 🟨 Add a repository URL  
-The URL can be the _doi_.
+The URL can be the link to the paper, for example via an identifier system like the doi, EuropePMC or PubMed.
```

Comment:
I'm not sure this fully clarifies what is meant. I don't think confusion is about whether the link is through DOI, PubMed, EuropePMC or even directly the journal, but rather **what** it links to. What about "The URL links to a publication/pre-print/website where the model is announced, introduced or used."?

Reply:
I thought I had already pushed the fix - sorry for the delay. I hope the [new commit](https://github.com/MetabolicAtlas/standard-GEM/pull/26/commits/60448c5202aa22f85f51745689c10039a80f0007) addresses the issue.